XMonad/Hooks/StatusBar.hs

```diff
@@ -74,22 +77,25 @@ import XMonad.Hooks.StatusBar.PP
 -- >    import XMonad.Hooks.StatusBar
 -- >    import XMonad.Hooks.StatusBar.PP
 --
--- The recommended way to use this module with xmobar, as well as any other
+-- The easiest way to use this module with xmobar, as well as any other
 -- status bar that supports property logging (you can read more about X11
 -- properties
 -- [here](https://en.wikipedia.org/wiki/X_Window_System_core_protocol#Properties)
 -- or
 -- [here](https://tronche.com/gui/x/xlib/window-information/properties-and-atoms.html),
 -- although you don't have to understand them in order to use the functions
--- below), is to use 'statusBarProp' with 'makeStatusBar', which takes care of
--- the necessary plumbing-no shell scripting required!
+-- below), is to use 'statusBarProp' with 'makeStatusBar'', which takes care of
+-- the necessary plumbing:
 --
 -- > main = do
--- >   mySB <- statusBarProp "xmobar" (pure myPP)
--- >   xmonad =<< makeStatusBar mySB myConf
+-- >   mySB <- statusBarProp "xmobar" (pure xmobarPP)
+-- >   xmonad =<< makeStatusBar' mySB def
```

Comment:
I think this is `makeStatusBar` , not `makeStatusBar'` (the latter takes an additional keybinding)

Reply:
Oh. The change to `makeStatusBar'` is intentional, I wanted the basic example to not rely on any custom `myPP` and `myConf`, but I forgot about the additional parameter, and now I see that `toggleStrutsKey` is still in `X.H.DynamicLog` so I can't easily use this. :-/

Reply:
9a4bfee205059f4f3d21d5d710f441fa615b3887, perhaps?

Reply:
Looks good :+1: